snorkel/db_helpers.py

```diff
@@ -37,7 +37,7 @@ def reload_annotator_labels(session, candidate_class, annotator_name, split, fil
 
         # Assemble query and check
         candidate_query = session.query(candidate_class)
-        for k, v in candidate_args.iteritems():
+        for k, v in candidate_args.items():
```

Comment:
In Python 2.7 this will materialize the whole list of tuples rather then creating a generator (correct?), which is problematic